src/gimelstudio/core/node/node.py

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------
 
+import os.path
```

Comment:
It might be worth looking into using [pathlib](https://docs.python.org/3/library/pathlib.html) instead of ``os.path``

Reply:
Actually, you are correct. I recently noticed that the loading still doesn't work correctly. It is based on the cwd for some reason.

Reply:
Also, loading from a directory inside the application itself won't work because when it is generated an an executable, the ``corenodes`` directory doesn't exist (as per one of my recent tests).

Perhaps putting all the nodes (including core nodes) in the ``nodescripts`` directory is better. Now that I think about it, that may be the best thing to do anyway since each node is like a "plugin" and that would be a better setup for installing/uninstalling nodes via the Node Manager. 
The (possible) downside is that it allows the end-user to tamper with the nodes directly.